php/deprecated.sls

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# vim: ft=sls
+
+php-deprecated-in-v1.0.0-test-fail:
+  test.succeed_without_changes:
+    - name: |
+        ################################################################################
+        #                                                                              #
+        #                   WARNING: BREAKING CHANGES SINCE `v1.0.0`                   #
+        #                                                                              #
+        ################################################################################
+        #                                                                              #
+        # Prior to `v1.0.0`, this formula provided two methods for managing php; the   #
+        # old method under `php` and the new method under `php.ng`. The old method     #
+        # has now been removed and `php.ng` has been promoted to be `PHP` in its       #
+        # place.                                                                       #
+        #                                                                              #
+        # If you are not in a position to migrate, please pin your repo to the final   #
+        # release tag before `v1.0.0`, i.e. `v0.37.1`.                                 #
+        #                                                                              #
+        # To migrate from `php.ng`, simply modify your pillar to promote the entire    #
+        # section under `php:ng` so that it is under `php` instead. So with the        #
+        # editor of your choice, highlight the entire section and then unindent one    #
+        # level. Finish by removing the `ng:` line.                                    #
+        #                                                                              #
+        # To migrate from the old `php`, first convert to `php.ng` under `v0.37.0`     #
+        # and then follow the steps laid out in the paragraph directly above.          #
+        #                                                                              #
+        ################################################################################
+    # - failhard: True
```

Comment:
do we need to keep this comment?

Reply:
As I mentioned earlier, it can be removed.  The reason I suggested leaving it there as a comment is because we'll be re-enabling it when we perform the actual deprecation.  Either way is fine, though.

Reply:
> As I mentioned earlier, it can be removed. The reason I suggested leaving it there as a comment is because we'll be re-enabling it when we perform the actual deprecation. Either way is fine, though.

OK, I see. :+1: